### PR TITLE
[Doc] linalg.ldl_factor: render the Shape of tensor A

### LINKED ISSUE #96864: [doc] linalg.ldl_factor : Shape of tensor `A` is rendered incorrectly
### 📚 The doc issue

image

Ref: https://pytorch.org/docs/master/generated/torch.linalg.ldl_factor.html?highlight=ldl_solve

### Suggest a potential alternative/fix

Shape for `A` should render as `(*, n, n)`.

cc @svekars @carljparker @jianyuh @nikitaved @pearu @mruberry @walterddr @IvanYashchuk @xwang233 @Lezcano

### PR DESCRIPTION
**Pull request by isdanni**
Summary: Fix: #96864

Test Plan: Please see GitHub tests.

### CODE REVIEW

**Review by lezcano — CHANGES_REQUESTED** [review]
Please, see the format that's used in the rest of linalg. In particular, we avoid using `:math:` as it takes quite a long time to render. Use single backticks instead.

File: torch/linalg/__init__.py [hunk 1]
```diff
@@ -871,7 +871,8 @@
 """ + r"""
 
 Args:
-    A (Tensor): tensor of shape (*, n, n) where * is zero or more batch dimensions consisting of symmetric or Hermitian matrices.
+    A (Tensor): tensor of shape `(*, n, n)` where `*` is zero or more batch dimensions
+                    consisting of symmetric or Hermitian matrices.
                     `(*, n, n)` where `*` is one or more batch dimensions.
```
Comment (lezcano) [inline torch/linalg/__init__.py:876]: Feel free to delete this line as well, as it's repeated. Same in the other function.

Comment (lezcano) [reply torch/linalg/__init__.py:876]: Also, this line has extra indentation. Same in the other one

**Review by lezcano — APPROVED** [review]
Thank you!

**Comment by isdanni** [comment]
@lezcano @IvanYashchuk any plans to merge this? :)

**Comment by lezcano** [comment]
You can merge it yourself writing `@pytorchbot merge` in a comment  :)

**Comment by lezcano** [comment]
Since you imported this, you should import the rebased branch onto phabricator as well, I believe.

**Pull request closed without merging**